js/age.js

```diff
@@ -1 +1,20 @@
+let age = 0
 
+if (age < 1 && age >= 1) {
+    console.log("You are an infant!")
+}
+else if (age = 2 && age <= 3){

```

Comment:
for our comparison operators in JS we use == or === for loosely equal and strictly equals ( best practice is to use strictly === becuase it also checks the type of the data so '2' == 2- true but  '2'===2 because string and number types